app/src/main/res/layout/item_selection_interaction_items.xml

```diff
@@ -0,0 +1,31 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto">
+  <data>
+    <variable
+      name="viewModel"
+      type="org.oppia.app.player.state.itemviewmodel.SelectionContentViewModel"/>
+  </data>
+  <RelativeLayout
+    android:id="@+id/checkbox_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="4dp"
+    android:orientation="vertical">
+    <androidx.appcompat.widget.AppCompatCheckBox
+      android:id="@+id/item_selection_checkbox"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:padding="4dp"
+      android:clickable="false"
```

Comment:
Here & for the next layout: why set clickable to false?

Reply:
As I am adding click to the layout container I made this clickable false.